.github/workflows/winget-submission.yml

```diff
@@ -6,8 +6,17 @@ on:
     types: [published]
 
 jobs:
+  sync-repo:
+    name: Sync winget-pkgs repo
+    runs-on: ubuntu-latest
+    steps:
+      - run: gh repo sync cinnamon-msft/winget-pkgs -b master
```

Comment:
Nit: Add a comment or Constant for this?

Reply:
@EricJohnson327, is it possible to add a variable to the repository?
`WINGET_PKGS_REPO="cinnamon-msft/winget-pkgs"`

Reply:
I've been thinking about this and I'm not sure we want to go the variable route right now.  It would make it more confusing for anyone else to come and make changes or fix things if it broke.  Single source of truth is best :)